internal/core/core_test.go

```diff
@@ -2204,6 +2204,59 @@ func TestAttributesInconsistentLength(t *testing.T) {
 	require.ErrorContains(t, err, errmsg, err)
 }
 
+func TestAttributesAllFill(t *testing.T) {
+	const above = float32(0)
+	const below = float32(0)
+	const stepsize = float32(4)
+	targetAttributes := []string{"samplevalue", "min"}
```

Comment:
Is there any specific reason that we are only testing `samplevalue` and `min` attributes? If not, would it be nicer to check all the attributes here?

Reply:
There is no real point in testing more.
Test makes sure that if you supply only fillvalue values, all works nicely and you get fillvalues in return.

As we do not have harsh testing requirements, we try to implement only tests that give best value - balance between actually having functionality tested and not overblowing test suite too much.

This is a whitebox test, we know the system, so we can use our knowledge that all attributes are actually implemented similarly and used attributes should have no meaning for the test outcome. The number of attributes shouldn't technically matter as well, but due to having to fill the same result for every attribute we better test on >1. Minimal necessary number is 2.

Hence I just chose two example attributes here and they happened to be samplevalue and min.